src/Compilers/CSharp/Portable/Symbols/TypeParameterSymbol.cs

```diff
@@ -436,6 +455,11 @@ internal bool IsReferenceTypeFromConstraintTypes(ImmutableArray<TypeSymbolWithAn
             return AnyConstraintTypes(constraintTypes, inProgress, (type, arg) => ConstraintImpliesReferenceType(type.TypeSymbol, arg));
         }
 
+        internal bool IsNotNullableIfReferenceTypeFromConstraintTypes(ImmutableArray<TypeSymbolWithAnnotations> constraintTypes, ConsList<TypeParameterSymbol> inProgress)
```

Comment:
>IsNotNullableIfReferenceTypeFromConstraintTypes [](start = 22, length = 47)

Consider using `Is` rather than `IsNot` for property and method names.

Reply:
> Consider using Is rather than IsNot for property and method names.

Do you have any specific suggestion in mind?

---
In reply to: [211449823](https://github.com/dotnet/roslyn/pull/29407#discussion_r211449823) [](ancestors = 211449823)

Reply:
`GetIsNullableIfReferenceType`, `IsNullableIfReferenceTypeFromConstraintTypes`, etc.

---
In reply to: [211678493](https://github.com/dotnet/roslyn/pull/29407#discussion_r211678493) [](ancestors = 211678493,211449823)

Reply:
> GetIsNullableIfReferenceType, IsNullableIfReferenceTypeFromConstraintTypes, etc.

I think these names with inverted value will be confusing because inverted value really means could be either (nullable or not nullable reference type). I will keep the current name for now. 

---
In reply to: [211684234](https://github.com/dotnet/roslyn/pull/29407#discussion_r211684234) [](ancestors = 211684234,211678493,211449823)